content/reinvent/_index.md

```diff
@@ -1,12 +1,47 @@
 ---
 title: Pulumi + AWS re:Invent 2024
-meta_desc: "Automate, secure and manage AWS infrastructure with IaC, secrets management, and cloud insights. Book 1:1s, join workshops, and transform your cloud strategy."
+meta_desc: "Visit Pulumi at booth 370 to book 1:1s, join workshops, and transform your cloud strategy. AWS re:Invent 2024 | December 2–6, 2024"
 meta_image: /images/reinvent/reinvent-meta-24.png
 type: page
 layout: reinvent
 aliases:
   - aws-reinvent-2021
 
+{
```

Comment:
@shughes26 I'm assuming the below code gets added to the <head> section of the page; if not, please add or advise how to do so.